lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -677,6 +677,10 @@ MITIGATION_01_IDENTITY_START_PAGE:
   events:
     next:
       targetState: MITIGATION_01_CRI_DCMAW
+      checkIfDisabled:
+        dcmaw:
+          targetJourney: TECHNICAL_ERROR
+          targetState: ERROR_NO_TICF
```

Comment:
I don't think we want to skip TICF in these scenarios(?)

Reply:
@Joe-Edwards-GDS Wasn't sure on this either - F2F uses this error sate, but maybe we want to use this error state/journey instead?
```
      targetJourney: TECHNICAL_ERROR
      targetState: ERROR
```

Reply:
The idea is that we should call TICF exactly once in each journey, as close to the end as possible.

In a F2F journey, we have to call it _before_ routing to the F2F CRI, so any errors that happen afterwards shouldn't call TICF again. (Similarly, with any errors that come from the TICF CRI itself)

In this case, I think we won't have called TICF, so we can just use the 'normal' error state :)